Sources/Sentry/SentryOptions.m

```diff
@@ -145,6 +145,9 @@ - (instancetype)init
         SentryHttpStatusCodeRange *defaultHttpStatusCodeRange =
             [[SentryHttpStatusCodeRange alloc] initWithMin:500 max:599];
         self.failedRequestStatusCodes = @[ defaultHttpStatusCodeRange ];
+        self.cacheDirectory
```

Comment:
`m`: You also need to add `cacheDirectory` to `validateOptions` and add tests to `SentryOptionsTest.m` similar to what we have for `release`.